### PR TITLE
Koa2 update

### PR DESCRIPTION
**Pull request by pgherveou**
updated to update to koa2 (ctx, next) middleware style
Can we push an alpha version like for the other middleware (e.g https://github.com/koajs/compress)

### CODE REVIEW

**Comment by pgherveou** [comment]
oups wrong branch target let me do it again

**Comment by pgherveou** [comment]
this should be merged into a v2.x branch actually

File: package.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
     "error.html"
   ],
   "devDependencies": {
-    "koa": "koajs/koa"
+    "koa": "^2.0.0-alpha.3"
```
Comment (coderhaoxin) [inline package.json]: 2.0.0 released :smile:

**Comment by coderhaoxin** [comment]
@pgherveou I'll review the below PRs first. And then rebase&merge this PR! :smile: 

https://github.com/koajs/error/pull/14
https://github.com/koajs/error/pull/13

**Comment by coderhaoxin** [comment]
@pgherveou can you rebase with master again? 😢

**Comment by coderhaoxin** [comment]
https://github.com/koajs/error/pull/24

**Pull request closed without merging**